webapp/src/locations.js

```diff
@@ -11,14 +11,13 @@ export const locations = {
   publish: '/:x/:y/publish',
   publishLand: (x, y) => `/${x}/${y}/publish`,
 
-  colorCodes: '/colorCodes',
-  privacy: '/privacy',
-
   parcel: '/:x/:y/detail',
   parcelDetail: (x, y) => `/${x}/${y}/detail`,
 
-  parcelTransfer: '/:x/:y/transfer',
-  parcelTransferDetail: (x, y) => `/${x}/${y}/transfer`,
+  activities: '/activities',
```

Comment:
`/activities` reminds me of [step brothers](https://www.google.com.ar/search?q=so+much+room+for+activities&tbm=isch) :P can we go with `/activity`?